### DISCUSSION TITLE
Problem with launching the quick start in the readme

### DISCUSSION THREAD

**Discussion by LAUJJL** [climateinteractive/SDEverywhere#458] [Q&A] [answered]
Hi . i am having a problem with the command npm create @sd... explained in the quick start from the readme. it gives me the message 
 Failed to install Emscripten SDK: Command failed with exit code 9009: python3 emsdk.py install 2.0.34
Python est introuvable. Ex�cutez sans argument pour proc�der � lCan anyone help me

**Comment by chrispcampbell** [comment]
Hi @LAUJJL, sorry for the delay, I was out of the office last week.  A couple other users have reported similar issues, please see my most recent reply in this [issue comment](https://github.com/climateinteractive/SDEverywhere/issues/460#issuecomment-2061971660) and let me know if you're able to get past this issue.

**Comment by LAUJJL** [comment]
Hi

Thank you for your answer.

I have spent several hours trying to solve the problem.
I have downloaded several time python and modified the settings puting the
python aliases to off, as mentioned in the issues
it stille says this:
 Failed to install Emscripten SDK: Command failed with exit code 1: python3
emsdk.py install 2.0.34
'python3' n'est pas reconnu en tant que commande interne
ou externe, un programme ex�cutable ou un fichier de commandes.

I have installed everything that needs to be installed.

Best regards;

Jean-Jacques Laublé

Le mer. 17 avr. 2024 à 21:14, Chris Campbell ***@***.***> a
écrit :

**Comment by chrispcampbell** [comment, marked as answer]
Hi @LAUJJL, my recommendation would be to install the Emscripten SDK manually instead of using the `@sdeverywhere/create` script to do it for you.  You can refer to my suggestions in [this comment](https://github.com/climateinteractive/SDEverywhere/issues/460#issuecomment-2062390362).  Be sure to activate a known working version, such as 2.0.34.  If you install it to an `emsdk` folder that is located in a parent folder above where you create your SDEverywhere project, then the SDEverywhere build tools should find it automatically.

**Comment by LAUJJL** [comment]
Hi

I will stop for the moment trying to use sdeverywhere. I should uninstall
everything node, python, git and SDK and maybe some others and install
again. I have lost too much time and sdeverywhere does not look like a
commercial software, but like a tool for specialists or researchers. I have
already made different experiences, trying to download the files published
by vensim for the web into my WIX web server. But I have to pay a WIX
developper to do it. I will try Forio and hope that it works; But it looks
like a commercial tool. If it does not work, i may go back to
sdeverywhere having no other solution.
Just one suggestion: change your quick start, so that it works without
having to make multiple changes, this is very confusing.

Anyhow, thank you for your help.

Best regards.

Jean-Jacques Laublé.

Le jeu. 18 avr. 2024 à 19:40, Chris Campbell ***@***.***> a
écrit :

**Comment by travisfranck** [comment]
@LAUJJL Thanks for trying SDE.  Unfortunately Windows hasn't been as well tested as macOS, but your efforts have helped the next person.  We really appreciate it. Let's us know how the rest of your "model-to-web" journey goes.

**Reply by LAUJJL** [reply]
Hi Travis

l seems that bringing SD to the net is not so easy. I tried Forio lately, but the documentation is terribly messy. One solution is to use insightmaker, but I am very used to Vensim. I have already a web site, But the model is not in line and must be downloaded with the vensim reader installer or the user can download the vensim reader from Vensim directly plus a powerpoint file that teels how to use the vensim reader, plus the models. I will keep this formula for the moment. One problem is that there is presently not any tool to create a user interface currently supported by Vensim like the old Venapps. There is one but in beta version, without any diocumentation, no examples and not readeable by the vensim reader; I do not understand Vensim's strategy. I thought that puting my model directly readable in the web site, would add a plus. But is it really the case? The better is often the ennemy of the good. 

Thank you for your interest

Jean-Jacques Laublé alias JJ

**Reply by LAUJJL** [reply]
Hi Travis
I did not try Stella. But inside maker could do the same thing for me, my models being very simple, to be understood by stakeholders. I prefer to stay with Vensim that I knkow quite well. If I see later on, that my models need to be published on the web. I will reconsider my strategy, but so far i do not know if I am going to interest stakeholders by the way I  present problems. It is possible that I do not interest  decision takers anyhow, and i will see that even if I do not publish and ask people to download files. By the way, the main problem of SD is to convince people to act, to my opinion. And I may find out, that this is true whatever the way you present the problem. 

Best regards; 


JJ
